src/components/molecules/modal.tsx

```diff
@@ -49,16 +74,17 @@ export const Modal = ({ children, className, hideCloseButton, name }: ModalProps
 						className={modalClasses}
 						exit="hidden"
 						initial="hidden"
+						ref={modalRef}
 						variants={modalVariants}
 					>
-						{hideCloseButton ? null : (
+						{!hideCloseButton ? (
```

Comment:
please return as it was.
The condition shouldn't be with `!`.

Reply:
reverted